packages/vscode-extension/src/webview/components/DeviceSettingsDropdown.tsx

```diff
@@ -330,4 +331,57 @@ const CameraItem = () => {
   );
 };
 
+const VolumeItem = () => {
+  const { project } = useProject();
+
+  const handleVolumeIncreaseDown = () => {
+    project.runCommand("RNIDE.deviceVolumeIncreaseButtonDown");
+  };
+
+  const handleVolumeIncreaseUp = () => {
+    project.runCommand("RNIDE.deviceVolumeIncreaseButtonUp");
+  };
+
+  const handleVolumeDecreaseDown = () => {
+    project.runCommand("RNIDE.deviceVolumeDecreaseButtonDown");
+  };
+
+  const handleVolumeDecreaseUp = () => {
+    project.runCommand("RNIDE.deviceVolumeDecreaseButtonUp");
+  };
+
+  // Make sure buttons get unpressed on unmount
+  React.useEffect(() => {
+    return () => {
+      handleVolumeDecreaseUp();
+      handleVolumeIncreaseUp();
+    };
+  }, []);
+
+  return (
+    <div className="dropdown-menu-item">
+      <span className="codicon codicon-unmute" />
+      Volume
+      <div className="volume-controls">
+        <button
+          title="Volume Down"
+          className="volume-button"
+          onMouseDown={handleVolumeDecreaseDown}
+          onMouseUp={handleVolumeDecreaseUp}
+          onMouseLeave={handleVolumeDecreaseUp}>
+          <span className="codicon codicon-chevron-down" />
+        </button>
+        <button
+          title="Volume Up"
+          className="volume-button"
+          onMouseDown={handleVolumeIncreaseDown}
+          onMouseUp={handleVolumeIncreaseUp}
+          onMouseLeave={handleVolumeIncreaseUp}>
+          <span className="codicon codicon-chevron-up" />
```

Comment:
```suggestion
          <span className="codicon codicon-dash" />
        </button>
        <button
          title="Volume Up"
          className="volume-button"
          onMouseDown={handleVolumeIncreaseDown}
          onMouseUp={handleVolumeIncreaseUp}
          onMouseLeave={handleVolumeIncreaseUp}>
          <span className="codicon codicon-add" />
```
I think I'd prefer "+" and "-" here over the chevrons.

Reply:
Not sure if it's better but this is how it looks after the update (I've used `codicon-remove` instead of `codicon-dash` since `codicon-dash` is too short and imo doesn't look good next to with `codicon-add`).

<img width="889" height="747" alt="Screenshot 2025-07-18 at 14 07 32" src="https://github.com/user-attachments/assets/7a943af3-8ae4-4574-8ddd-1c3d71cd50bd" />

Reply:
I think it's better this way.
So far, I think we've used chevron icons exclusively for scrolling and drop-downs, so they looked out of place here for me.